flink-table/flink-table-planner-blink/src/test/resources/explain/testStatementSetExecutionExplain.out

```diff
@@ -32,9 +32,9 @@ LegacySink(name=[`default_catalog`.`default_database`.`MySink`], fields=[first])
     } ]
   }, {
     "id" : ,
-    "type" : "SourceConversion(table=[default_catalog.default_database.MyTable, source: [CsvTableSource(read fields: last)]], fields=[last])",
+    "type" : "SourceConversionToRowData",
     "pact" : "Operator",
-    "contents" : "SourceConversion(table=[default_catalog.default_database.MyTable, source: [CsvTableSource(read fields: last)]], fields=[last])",
+    "contents" : "SourceConversionToRowData",
```

Comment:
the same as the pr before, it would be better to have the info of table in operator name.

Reply:
revert the change